### PR TITLE
Feat.: uber image add default app for local storage.

### PR DESCRIPTION
**Pull request by olivershen-wow**
## Pull Request Checklist

- [X] Tests for the changes have been added (for bug fixes / features)
- [X] Code compiles correctly with all tests are passed.
- [X] I've read the [contributing guide](https://github.com/microsoft/HydraLab/blob/main/CONTRIBUTING.md#making-changes-to-the-code) and followed the recommended practices.
- [X] [Wikis](https://github.com/microsoft/HydraLab/wiki) or [README](https://github.com/microsoft/HydraLab/blob/main/README.md) have been reviewed and added / updated if needed (for bug fixes / features)

### Does this introduce a breaking change?
*If this introduces a breaking change for Hydra Lab users, please describe the impact and migration path.*

- [ ] Yes
- [X] No

## Pull Request Description

**A few words to explain your changes**:  
- Add new apk file to uber docker image and corresponding DB record. 
- Add sql script. 
- Change default fileLimitDay of LOCAL storage to -1, to disable expiry of file existing in Uber image record.

### How you tested it
*Please make sure the change is tested, you can test it by adding UTs, do local test and share the screenshots, etc.*
Test docker build locally, with the test package record displayed and run correctly.

Please check the type of change your PR introduces:
- [ ] Bugfix
- [X] Feature
- [ ] Technical design
- [ ] Build related changes
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Code style update (formatting, renaming) or Documentation content changes
- [ ] Other (please describe): 

### Feature UI screenshots or Technical design diagrams
*If this is a relatively large or complex change, kick it off by drawing the tech design with PlantUML and explaining why you chose the solution you did and what alternatives you considered, etc...*
image

### CODE REVIEW

**Review by zhou9584 — DISMISSED** [review]

File: README.md [hunk 1]
```diff
@@ -106,7 +106,39 @@ docker run -e STORAGE_TYPE=AZURE -e BLOB_CONNECTION_STR=${YOUR_BLOB_CONNECTION_S
...
+- For the `APK/IPA FILE` field, select the downloaded `record_release.apk` file.
+- For the `TEST APK/JAR/JSON FILE` field, select the downloaded `record_androidTest.apk` file.
+- Click `Upload` button to upload the apps.
+- Wait for the upload process to finish, then you can follow the same steps of `1.` to run the test.
+
+Notice: 
+- The existing package record in the page is preset only for LOCAL storage usage, and running directly with that record would lead to error. 
+- You need to upload the package again if you are using your own storage service.
+- Multiple storage types are not allowed to be used together in one service, and this specific situation is happening only in Uber image where context and database has been predefined assuming default storage type is LOCAL for onboarding users.
+
```
Comment (hydraxman) [inline README.md:139]: Could you use new bing or chatGPT to optimize the wording of this write-up?

File: center/src/main/resources/application.yml [hunk 2]
```diff
@@ -55,7 +55,7 @@ app:
     type: ${STORAGE_TYPE:LOCAL} # current available options: LOCAL (default), AZURE
     azure:
       connection: ${BLOB_CONNECTION_STR:}
-      fileLimitDay: 6
+      fileLimitDay: ${FILE_LIMIT_DAY:6}
```
Comment (hydraxman) [inline center/src/main/resources/application.yml:58]: Suggest change a better name for this property: fileExpiryDay cc @zhou9584

File: README.md (hunk 1 above)
Comment (olivershen-wow) [reply README.md:139]: Optimized with GPT

File: center/src/main/resources/application.yml (hunk 2 above)
Comment (olivershen-wow) [reply center/src/main/resources/application.yml:58]: Resolved for now. Will update in another PR specifically.

File: README.md [hunk 3]
```diff
@@ -106,7 +106,39 @@ docker run -e STORAGE_TYPE=AZURE -e BLOB_CONNECTION_STR=${YOUR_BLOB_CONNECTION_S
 
 Enjoy starting your journey of exploration!
 
-**Note: Uber now only provides the Espresso test feature for Android, please refer to this section for more features: [For Hydra Lab User](#for-user)**
+**Step 4. try the test procedure with simple setup**
```
Comment (hydraxman) [inline README.md:109]:
```suggestion
**Step 4. Perform the test procedure with a minimal setup**
```

image

File: README.md [hunk 4]
```diff
@@ -106,7 +106,39 @@ docker run -e STORAGE_TYPE=AZURE -e BLOB_CONNECTION_STR=${YOUR_BLOB_CONNECTION_S
...
+
+Note: Uber image now only provides the **Espresso/Instrumentation test** feature for Android, please refer to the links within "User manual" section of this page for more features: [Hydra Lab Wikis](https://github.com/microsoft/HydraLab/wiki).
```
Comment (hydraxman) [inline README.md:111]: image

Note: For Android, Uber image only supports Espresso/Instrumentation test. See “User manual” section on this page for more features.

File: README.md (hunk 3 above)
Comment (olivershen-wow) [reply README.md]: Applied, thx

File: README.md (hunk 4 above)
Comment (olivershen-wow) [reply README.md]: Applied, thx

File: center/Dockerfile [hunk 5]
```diff
@@ -51,6 +53,9 @@ COPY ${STARTUP_FOLDER_PATH}/nginx.conf /etc/nginx/
 COPY ${STARTUP_FOLDER_PATH}/restartMonitorService.sh restartMonitorService.sh
 COPY ${STARTUP_FOLDER_PATH}/splitLog.sh splitLog.sh
 COPY ${STARTUP_FOLDER_PATH}/start.sh start.sh
+COPY uber_image/${CENTER_INIT_SQLITE} /hydra/data/${CENTER_INIT_SQLITE}
+COPY uber_image/record_release.apk /storage/local/pkgstore/2023/04/23/record_release.apk
```
Comment (hydraxman) [inline center/Dockerfile:57]:
```suggestion
COPY uber_image/hydra_lab_android_client_release.apk /storage/local/pkgstore/2023/04/23/hydra_lab_android_client_release.apk
```
I suggest we use new name for this apk. Also apply to the test app.

Comment (olivershen-wow) [reply center/Dockerfile]: Update the dest file name. Source file would be copied from Common module.

**Pull request merged**